Index: lib-src/libnyquist/nyquist/xlisp/xlbfun.c
===================================================================
--- lib-src/libnyquist/nyquist/xlisp/xlbfun.c.orig	2017-03-03 21:27:17.000000000 +0200
+++ lib-src/libnyquist/nyquist/xlisp/xlbfun.c	2017-03-12 10:16:53.207654852 +0200
@@ -603,7 +603,10 @@ LVAL xcleanup(void)
 {
     xllastarg();
     xlcleanup();
-    /* compiler might (wrongly) complain there is no return value */
+    /* this point will never be reached because xlcleanup() does a
+       longjmp(). The return is added to avoid false positive
+       error messages from static analyzers and compilers */
+    return (NIL);
 }
 
 /* xtoplevel - special form 'top-level' */
Index: src/PluginManager.h
===================================================================
--- src/PluginManager.h.orig	2017-03-03 21:27:17.000000000 +0200
+++ src/PluginManager.h	2017-03-12 10:16:53.207654852 +0200
@@ -167,7 +167,6 @@ typedef std::map<PluginID, PluginDescrip
 
 typedef wxArrayString PluginIDList;
 
-class ProviderMap;
 class PluginRegistrationDialog;
 
 class PluginManager final : public PluginManagerInterface
